package.json

```diff
@@ -1,30 +1,42 @@
 {
-  "name"        : "hexo-uncss",
-  "version"     : "0.1.4",
-  "description" : "uncss plugin for Hexo.",
-  "keywords"    : [ "hexo", "hexo-plugin", "hexo-filter", "uncss", "css", "html" ],
-  "homepage"    : "https://github.com/vseventer/hexo-uncss",
-  "bugs"        : "https://github.com/vseventer/hexo-uncss/issues",
-  "license"     : "MIT",
-  "author"      : "Mark van Seventer <mark@vseventer.com>",
-  "repository"  : { "type": "git", "url": "vseventer/hexo-uncss" },
-  "main"        : "./index.js",
-  "scripts"     : {
-    "pretest" : "./node_modules/.bin/jshint --reporter=./node_modules/jshint-stylish lib/ test/ *.js",
-    "test"    : "./node_modules/.bin/mocha test/"
+  "name": "hexo-uncss",
+  "version": "0.1.4",
+  "description": "uncss plugin for Hexo.",
+  "keywords": [
+    "hexo",
+    "hexo-plugin",
+    "hexo-filter",
+    "uncss",
+    "css",
+    "html"
+  ],
+  "homepage": "https://github.com/vseventer/hexo-uncss",
+  "bugs": "https://github.com/vseventer/hexo-uncss/issues",
+  "license": "MIT",
+  "author": "Mark van Seventer <mark@vseventer.com>",
+  "repository": {
+    "type": "git",
+    "url": "vseventer/hexo-uncss"
+  },
+  "main": "./index.js",
+  "scripts": {
+    "pretest": "./node_modules/.bin/jshint --reporter=./node_modules/jshint-stylish lib/ test/ *.js",
+    "test": "./node_modules/.bin/mocha test/"
   },
   "dependencies": {
-    "bluebird"        : "3.1.x",
-    "debug"           : "2.2.x",
-    "minimatch"       : "3.0.x",
-    "object-assign"   : "4.0.x",
-    "stream-to-array" : "2.2.x",
-    "uncss"           : "0.12.x"
+    "bluebird": "3.5.1",
+    "debug": "3.1.0",
+    "minimatch": "3.0.4",
+    "object-assign": "4.1.1",
+    "stream-to-array": "2.3.0",
+    "uncss": "0.16.2"
```

Comment:
Hi, do you have the same issue without this PR?
I believe the issue you have is because uncss is executing jquery (the JS in your page) before executing uncss.
I do not have this issue because I do not have JS on my pages

Reply:
Nope, I don't have this issue without this PR (I have errors coming from the js in <script></script> tags though)

Reply:
This line: https://github.com/Kakise/hexo-uncss-stylus/blob/35668fdc8ba765fb4b506d308daa6ee5005708a9/index.js#L45 seems to fix the problem with both <script></script> and js files but the filter isn't executed automatically anymore. I don't know how to fix this.  It also fixes this issue: https://github.com/Kakise/Blog/issues/24 but well, you have to specify the files manually.

Reply:
I'l not sure what the "phases of generations" are, but semanticaly speaking, this seems like a wrong fix (playing with the CSS has not much to do with the HTML)

And what happens if you change priority without changing the filter?
so if you decrease priority (>10) ?

Reply:
Same result unfortunately